about.html

```diff
@@ -0,0 +1,170 @@
+      rel="stylesheet"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <link rel="stylesheet" href="./css/general.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/about.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <div class="nav-top">
+      <div class="container">
+        <ul>
+          <a href="#">
+            <i class="fa-brands fa-facebook-f"></i>
+          </a>
+          <a href="#">
+            <i class="fa-brands fa-twitter"></i>
+          </a>
+          <a href="#">English</a>
+          <a href="#">MyPage</a>
+          <a href="#">Logout</a>
+        </ul>
+      </div>
+    </div>
+    <nav>
+      <div class="container">
+        <ul class="navbar">
+          <a href="index.html" class="logo"
+            ><img src="./images/cc-global.png" alt=""
+          /></a>
+
+          <ul class="nav-items">
+            <a href="about.html">About</a>
+            <a href="about.html">Join</a>
+            <a href="about.html">Program</a>
+            <a href="about.html">News</a>
+            <a href="#">CC Campaign</a>
+          </ul>
+        </ul>
+      </div>
+    </nav>
+
+    <nav class="mobile-menu">
+      <span class="close">X</span>
+
+      <ul class="nav-items">
+        <a href="about.html">About</a>
+        <a href="#">Join</a>
+        <a href="#program">Program</a>
+        <a href="#partners">Partners</a>
+        <a href="#speakers">Speakers</a>
+        <a href="#">CC Campaign</a>
+      </ul>
+    </nav>
+
+    <section class="about landing">
+      <div class="container">
+        <div class="menu-icon">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="landing-content">
+          <span class="sub-heading">"Hello! Sharing World"</span>
+          <h1 class="primary-heading">Creative Commons Global Summit 2015</h1>
+          <p class="landing-text">
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years. This year, we hope
+            to expand our invitation list including organizations and
+            individuals who want to work with us on shared projects that advance
+            the cause of the Commons, free culture and open knowledge.
+          </p>
+          <p class="content">
+            Please contact via our email if you have any further questions about
+            CC Global Summit 2015!
+          </p>
+          <a class="email">thisorganization@email.com</a>
+        </div>
```

Comment:
- [ ] Kindly fix this overlapping text in mobile screen in about page as shown in the following image 
>![image](https://user-images.githubusercontent.com/53402600/233788892-d8648864-8adf-4ef3-8dd8-02bf5e8a5398.png)

Reply:
- [ ] Again here as requested by the previous reviewer, kindly fix the overlapping text in mobile view of the `about` page
![image](https://user-images.githubusercontent.com/81932478/233792607-1261702f-001c-46ce-9c5f-a1c91ddf3638.png)

Reply:
it works on my side

kindly, check this video

https://www.loom.com/share/de52d6e9c6094ecebf0f4596a8046e12